README.md

```diff
@@ -140,7 +140,9 @@ Status: 200 OK
 
 ```json
 {
-  "text": "The API doesn't require an access token to share this message."
+  "text": "The starter API doesn't require an access token to share this public message.",
+  "api": "api_django_python_hello-world",
+  "branch": "starter"
```

Comment:
I know this is only the README, but `api` and `branch` now go under the `metadata` property 🙂